ci-operator/config/openshift/special-resource-operator/openshift-special-resource-operator-master.yaml

```diff
@@ -87,6 +86,54 @@ tests:
   commands: make verify
   container:
     from: src
+- as: disconnected-build
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.10"
+  run_if_changed: ^Dockerfile$
+  steps:
+    test:
+    - as: create-podman-container
+      cli: latest
+      commands: |
+        oc -n openshift-config  get secret/pull-secret -o yaml | sed 's/openshift-config/default/' | sed 's/\.dockerconfigjson/auth.json/' | grep -v type: | oc apply -f -
+        oc -n default apply -f ci/podman.yaml
+        oc -n default wait pod/podman --for=condition=Ready
+      from: src
+      resources:
+        requests:
+          cpu: 300m
+          memory: 500Mi
+    - as: copy-sro-to-podman-container
+      cli: latest
+      commands: |
+        tar -cf sro.tar *
+        oc -n default exec pod/podman -- mkdir /tmp/sro
+        oc cp sro.tar default/podman:/tmp/sro/
+        oc -n default exec pod/podman -- tar -C /tmp/sro/ -xf /tmp/sro/sro.tar
+        oc -n default exec pod/podman -- rm /tmp/sro/sro.tar
+      from: src
+      resources:
+        requests:
+          cpu: 300m
+          memory: 500Mi
+    - as: build-sro-disconnected
+      cli: latest
+      commands: |
+        oc -n default exec pod/podman -- podman pull \
+            --authfile /run/user/0/containers/auth.json \
```

Comment:
Me too but this in not the case, the default location is based on `XDG_RUNTIME_DIR` as mentioned in the [podman doc](https://docs.podman.io/en/latest/markdown/podman-login.1.html#authfile-path) which is set to `/run/user/0` on my machine but empty inside the podman pod.

We can set this env in the yaml deploying podman but I don't think it is clear that podman is using `XDG_RUNTIME_DIR`, setting the `authfile` seems much more clear IMO.

Reply:
I am fine with using `--authfile`, but then I don't think we should put the file into such a special directory then. Maybe you want to merge this now and revisit this later?

Reply:
Yes, I will like to merge this one. Anyway it doesn't really matter where we put it, just though it will be better to put it in the "expected" location.